### ISSUE TITLE
Music app unable to play music from sd-card

### ISSUE THREAD

**Issue by LlamasJM** [ubports/ubuntu-touch#1134]
### Description of the feature
In Frieza, Music App is unable to play music from sd-card. Doesn't recognize album or artist. I can play the songs with the Video app, b
![screenshot20200419_173028196](https://user-images.githubusercontent.com/35621579/79692206-2823d980-8264-11ea-9a34-249d8518be4b.png)
![screenshot20200419_173108708](https://user-images.githubusercontent.com/35621579/79692209-2c4ff700-8264-11ea-94ad-454df6fbe053.png)
![screenshot20200419_173117802](https://user-images.githubusercontent.com/35621579/79692213-2f4ae780-8264-11ea-8ada-994a305db91e.png)
![screenshot20200419_173124388](https://user-images.githubusercontent.com/35621579/79692218-32de6e80-8264-11ea-948d-99f7b9fd0ca9.png)

ut not with Music app. 

### Illustrations

- Device: Frieza
- Channel: Rc
- Build: Ubuntu 16.04 (OTA-12 rc)

### Steps to reproduce
Music app unable to play music files from sd-card

### Expected behavior
Music files from sd-card should appear and be played.

### Actual behavior
From sd-card, no artist or album appears. Everything is "unknown artist", and every song is only a number, or a title, without length.

**Comment by advocatux** [comment]
Hi, have you checked that the sd-card is OK? In which format is it?

**Comment by LlamasJM** [comment]
Yes, it's OK. I've changed the Tablet OS (originally, Ubuntu Touch Tablet) sometimes, but the sd-card remains the same, with the same format. "Files" app recognizes perfectly card and music, "Video" app import and reproduce songs, but "Music" app can't import or reproduce songs from Sd-Card. Strange.

**Comment by advocatux** [comment]
You can try to remove Music app's `cache`, `config`, and `data` files using UT Tweak tool for example, then reboot your device, and then open Music app. That should (re)index your music collection.

**Comment by Flohack74** [comment]
Can it be that the music-app simply cannot handle SD cards? I never tried this...

**Comment by advocatux** [comment]
@Flohack74 no, music-app has no problem handling a SD card :)

**Comment by LlamasJM** [comment]
Issue is evolving... 

Same SD-Card on Bq Aquaris 4.5 (Ubuntu Edition), Music app recognize music perfectly: album, artist...

Today, on Frieza, recognize songs from SD-Card, but neither Album, nor Artist. All my music is "Unknown Artist - Unknown Album". Probably next week Music app will recognize everything, ho ho...

**Comment by Flaburgan** [comment]
I cannot reproduce with a Fairphone 2 OTA-9: the music app can find songs in my SD card.
